src/main/controllers/helpers/YourDetailsAnswersHelper.ts

```diff
@@ -159,27 +167,52 @@ export const getYourDetails = (
         ],
       },
     },
-    {
-      key: {
-        text: translations.personalDetails.languageLabel,
-        classes: 'govuk-!-font-weight-regular-m',
-      },
-      value: {
-        text:
-          userCase.claimantContactLanguagePreference === EnglishOrWelsh.ENGLISH
-            ? translations.personalDetails.english
-            : translations.personalDetails.welsh,
-      },
-      actions: {
-        items: [
+    ...(userCase.caseTypeId === CaseTypeId.SCOTLAND
```

Comment:
This is very hard to read,  this applies to the entire massive return statement not just your addition. Perhaps the whole thing should be refactored to be more readable. Not going to block this though

Reply:
Definitely a lot to clean up here, after talking privately I think we'll let it slide and clean this whole file later.